### ISSUE TITLE
Security problems. Many critical vulnerabilities

### ISSUE THREAD

**Issue by ISnotes** [NginxProxyManager/nginx-proxy-manager#3503]
**Checklist**
- Have you pulled and found the error with `jc21/nginx-proxy-manager:latest` docker image?
  - **Yes** / No
- Are you sure you're not using someone else's docker image?
  - **Yes** / No
- Have you searched for similar issues (both open and closed)?
  - **Yes** / No

**Describe the bug**

Hello!
I checked for vulnerabilities through the containercve.com service. I was very surprised to see more than 400, many of them critical (>9.0). Please correct.

**Nginx Proxy Manager Version**

2.11.1

**To Reproduce**
1. Go to https://containercve.com
2. Paste "jc21/nginx-proxy-manager:2.11.1" and click "Scan"
3. See report

**Expected behavior**

**Screenshots**

![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/12543500/fd2c45c4-6db3-4c36-8335-a26500d6c65c)

**Operating System**

**Additional context**

**Comment by JustinBack** [comment]
You gotta keep in mind that some of these vulnerabilities are not applicable most of the time e.g. the container has the lib installed but it's not used.

glib for instance notes:
> NOTE: Upstream comments indicate "this is being treated as a non-security bug and no real threat.

**Comment by Shocktrooper** [comment]
Shouldn't these just be uninstalled then so that there aren't false positives?

**Comment by ISnotes** [comment]
It would be great to have a version with alpine as the base image

**Comment by ISnotes** [comment]
Alpine is used in most of the most popular projects, and suddenly it is "unreliable". Suspiciously

**Comment by AdamCzepiel78** [comment]
i cant use this image in production when there are so many vulnerabilities. 
Please dear @jc21 team, try to fix it

**Comment by Ramalama2** [comment]
I Agree that this image is unusable everywhere, where Security is very Important.
Additionally the image is far too big for what it is.

But both points are gonna be solved with v3 mostly. So the Future is looking great!
I don't think jc21 needs to waste his time here, as he is already working on it.

Cheers

**Comment by Ramalama2** [comment]
> > But both points are gonna be solved with v3 mostly. So the Future is looking great! I don't think jc21 needs to waste his time here, as he is already working on it.
> 
> Is this documented anywhere, or how do you know this information?

Searched here in the issues around and everything jc21 wrote. I had initially the same concerns, so i was interested either.
But dont remember now where exactly, i had to post links :-(

However, as far i've seen he isn't happy about the huge image, but don't want to use alpine linux either. I don't remember what he want to use tbh, but he mentioned it somewhere.

There is already a image, but not sure if its working, at least its half as small as the current v2 images:
https://hub.docker.com/layers/jc21/nginx-proxy-manager/v3/images/sha256-858a3be38a605b3af148d6eb42ff7bbbea668b51d7e9ad16294386a20c283f1f?context=explore
In the Changes of the v3 branch in github is mentioned that the Admin UI switches to Chakra UI...
It will be entirely go based or something like that, ugh i dont want to spread wrong information, i simply dont remember :-(

But it looks to me, like v3 will take still a long time.
Seems to be simply like a lot of work, and if a lot of work is involved the motivation suffers, so i dont expect anything soon.

Cheers

**Comment by BearTS** [comment]
Almost a year now, Bumping this up.
+1 for alpine image

**Comment by 7heMech** [comment]
What about ditching node and going Alpine and Bun hahah

**Comment by BearTS** [comment]
Ended up switching to pangolin given this thread is not at all being looked at 
@jc21
